requirements/requirements-test.txt

```diff
@@ -3,3 +3,5 @@ coverage==5.5
 pytest-cov==2.11.1
 pytest==6.2.3
 ruamel.yaml==0.17.4
+filelock>=3.0
+packaging>=15.0
```

Comment:
These requirements are in addition to the core `requirements.txt`. Only testing requirements should be listed here.